src/Chapter3.hs

```diff
@@ -907,9 +1048,26 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+
 class Append a where
     append :: a -> a -> a
 
+newtype Golden = Golden Int -- Golden instead of gold because Gold has been used already
+
+instance Append Golden where
+    append :: Golden -> Golden -> Golden
+    append (Golden a) (Golden b) = Golden(a + b)
+
+instance Append [a] where
+    append :: [a] -> [a] -> [a]
+    append a b = a ++ b
```

Comment:
You can eta-reduce the implementation of `append` for brevity. So you can see how easy it is to create instances :slightly_smiling_face: 
```suggestion
    append = (++)
```